### PR TITLE
Handle pending payout status in bounty

### PR DESCRIPTION
**Pull request by ekowalsk**
This PR adds new columns in bounty lists to render data for Pending Payout status

### CODE REVIEW

File: packages/page-bounties/src/Bounty.tsx [hunk 1]
```diff
@@ -102,11 +100,26 @@ function Bounty ({ bestNumber, bounty, className = '', description, index }: Pro
   );
 }
 
+const DueBlocks = ({ dueBlocks }: { dueBlocks: BN | undefined }): JSX.Element => {
```
Comment (jacogr) [inline packages/page-bounties/src/Bounty.tsx:103]: Always prefer the `function DueBlocks (...) ...` form, consistent with the rest of the apps UI.

Comment (ekowalsk) [reply packages/page-bounties/src/Bounty.tsx]: Corrected :)

**Comment by jacogr** [comment]
No need to force-push, we do squash commits - force pushing makes is a bit difficult once the branch is checked out for review. (Some extra hoops to jump through)

**Review by jacogr — APPROVED** [review]
Thank you.

**Comment by ekowalsk** [comment]
> No need to force-push, we do squash commits - force pushing makes is a bit difficult once the branch is checked out for review. (Some extra hoops to jump through)

Sorry, I will keep that in mind for future editing :)

**Comment by jacogr** [comment]
No issues, I know it is ingrained in a lot of people :)

**Pull request merged**